release-tools/filter-junit.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This command filters a JUnit file such that only tests with a name
```

Comment:
@mattcary can you changes to combine the junit files be refactored now that it's actually imported in the repo?

Reply:
#591

Great point. It may be a little complicated as IIRC I made some changes to the junit-filter code which was built only as a command line tool and not a go library. But we should be able to dedup something, maybe by also extending the refactoring to the release tools.